### PR TITLE
adding attendance, challenge and hacker rank exercises  from August 26

### PR DESCRIPTION
**Pull request by ClaudiaHG23**

### CODE REVIEW

**Review by luisod78 — APPROVED** [review]

File: ClaudiaHernandezGarcia/26Aug/HackerRank/inheritanceII/Arithmetic.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package inheritanceII;
+
+class Arithmetic {
```
Comment (luisod78) [inline ClaudiaHernandezGarcia/26Aug/HackerRank/inheritanceII/Arithmetic.java:3]: hmmmmmm
 ".....A class named Arithmetic with a method named add that takes 2 integers..."

**Review by luisod78 — CHANGES_REQUESTED** [review]

File: ClaudiaHernandezGarcia/26Aug/HackerRank/outputFormatting/Solution.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package outputFormatting;
+
+import java.util.Scanner;
+
+public class Solution {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("================================");
```
Comment (luisalanmz) [inline ClaudiaHernandezGarcia/26Aug/HackerRank/outputFormatting/Solution.java:8]: Put this String in a Constant, please

File: ClaudiaHernandezGarcia/26Aug/HackerRank/outputFormatting/Solution.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        for (int i = 0; i < 3; i++) {
+            String s1 = sc.next();
+            int x = sc.nextInt();
+
+            System.out.printf("%-14s %03d%n", s1, x);
+        }
+
+        sc.close();
+        System.out.println("================================");
```
Comment (luisalanmz) [inline ClaudiaHernandezGarcia/26Aug/HackerRank/outputFormatting/Solution.java:17]: Put this String in a Constant, please

File: ClaudiaHernandezGarcia/26Aug/HackerRank/inheritanceI/Animal.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package inheritanceI;
+
+public class Animal {
+    void walk() {
+		System.out.println("I am walking");
```
Comment (luisalanmz) [inline ClaudiaHernandezGarcia/26Aug/HackerRank/inheritanceI/Animal.java:5]: You could put the String "I am" in a Constant and then concatenate the action

**Review by luisalanmz — COMMENTED** [review]
a coupe of suggestions

File: ClaudiaHernandezGarcia/26Aug/HackerRank/inheritanceII/Arithmetic.java (hunk 1 above)
Comment (ClaudiaHG23) [reply ClaudiaHernandezGarcia/26Aug/HackerRank/inheritanceII/Arithmetic.java:3]: Solved

**Review by luisalanmz — APPROVED** [review]
LGTM

**Review by luisod78 — APPROVED** [review]

**Pull request merged**